configs/twins/twins.yml

```diff
@@ -36,7 +22,7 @@ Models:
   Config: configs/twins/twins_pcpvt-s_fpn_fpnhead_8x4_512x512_80k_ade20k.py
   Weights: https://download.openmmlab.com/mmsegmentation/v0.5/twins/twins_pcpvt-s_fpn_fpnhead_8x4_512x512_80k_ade20k/twins_pcpvt-s_fpn_fpnhead_8x4_512x512_80k_ade20k_20211201_204132-41acd132.pth
 - Name: twins_pcpvt-s_uperhead_8x4_512x512_160k_ade20k
-  In Collection: twins
+  In Collection: Twins-UPerNet
```

Comment:
what's the difference between upernet and twin-upernet?

Reply:
No difference, they are the same. It is cause by this line: https://github.com/MengzhangLI/mmsegmentation/blob/0dd5931e9142fc5f4a7284ead82d662631c8f5be/.dev/md2yml.py#L169

We could solve it by directly using `method = els[method_id]`.